plugins/mcreator-localization/help/default/potioneffect/cures.md

```diff
@@ -0,0 +1,7 @@
+This parameter controls the potion healing
+
+There are various types of cures:
+
+* PROTECTED_BY_TOTEM: When the totem protects you from death, cures any effect by default

```

Comment:
This tip should be corrected

Reply:
What should I correct?

Reply:
Terms like PROTECTED_BY_TOTEM still refer to dropdown values. Why would user need to see machine names here if not needed? 

Adapt texts to be more adapted to new UI

Reply:
Done